api/src/app.py

```diff
@@ -22,8 +26,12 @@
 
 if ENVIRONMENT == 'development':
     metrics = PrometheusMetrics(app)
+    app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///:memory:'
+    app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
 else:
     metrics = GunicornInternalPrometheusMetrics(app)
+    app.config['SQLALCHEMY_DATABASE_URI'] = DATABASE_DIR or os.path('data.db')
```

Comment:
This needs more validation. ideally we want to pass the default path as a default to `os.getenv()` further up, then we will also need to call `abspath` on it.